### ISSUE TITLE
Ability to add CORS settings for routes

### ISSUE THREAD

**Issue by svensklave** [ice-services/moleculer-web#10]
This custom code works but only for "GET" method:
This is part of route config
```
// Set CORS headers
onBeforeCall(ctx, route, req, res) {
  ctx.meta.headers = req.headers;
   res.setHeader('Access-Control-Allow-Origin', '*');
   res.setHeader('Access-Control-Request-Method', '*');
   res.setHeader('Access-Control-Allow-Methods', 'OPTIONS, GET, POST');
   res.setHeader('Access-Control-Allow-Headers', 'Content-Type');
   if ( req.method === 'OPTIONS' ) {
     res.writeHead(200);
     res.end();
     return;
     }
}
```
on POST method this does not work! (because of different logic in source code)

Would be nice to integrate CORS settings in some nice way
so we can easy add CORS settings for routes and also for aliased routes if possible

**Comment by icebob** [comment]
Proposal based on [rafanami gist](https://gist.github.com/rafanami/ea7cc3be56b2ec0dfeb698aa204c1a29):

```js
module.exports = {
    settings: {
        // Global CORS settings
        cors: {
            origins: ["*"],
            methods: ["GET", "POST", "PUT"],
            headers: []
        },
        routes: [
            {
                path: '/',
                
                // Route-specified CORS settings
                cors: {
                    origins: ["http://foo.example"],
                    methods: ["GET"], // overwrite the global settings
                    headers: ["Accept", "Content-Type"]
                }
            }
        ]
    }

};
```

**Comment by icebob** [comment]
Implemented in #16
```js
module.exports = {
    settings: {
        // Global CORS settings
        cors: {
            origin: "*",
            methods: ["GET", "OPTIONS", "POST", "PUT", "DELETE"],
            allowedHeaders: [],
            exposedHeaders: [],
            credentials: false,
            maxAge: 3600
        },
        routes: [
            {
                path: '/',
                
                // Route CORS settings
                cors: {
                    origin: ["http://localhost:3000", "https://localhost:4000"],
                    methods: ["GET", "OPTIONS", "POST"],
                    credentials: true
                },
            }
        ]
    }

};
```
